### PR TITLE
Docker configuration

### PR DESCRIPTION
**Pull request by holsoni**

### CODE REVIEW

**Review by JeniaSan — APPROVED** [review]
LGTM!

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,16 @@
+#Builder stage
+FROM openjdk:17-jdk-slim as builder
+WORKDIR application
+ARG JAR_FILE=target/*.jar
+COPY ${JAR_FILE} application.jar
+RUN java -Djarmode=layertools -jar application.jar extract
+
+#Final stage
```
Comment (JeniaSan) [inline Dockerfile:8]: Please, remove '#' lines

Comment (vovasalyha) [reply Dockerfile:8]: It's ok, you can keep

**Review by vovasalyha — APPROVED** [review]
GJ

**Pull request merged**